src/Command/CheckAllSchemaTemplatesNameFieldsCommand.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jobcloud\SchemaConsole\Command;
+
+use Jobcloud\SchemaConsole\Helper\SchemaFileHelper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CheckAllSchemaTemplatesNameFieldsCommand extends Command
+{
+    private const TYPES_FOR_VALIDATION = [
+        'record',
+        'enum',
+        'fixed'
+    ];
+
+    private const REGEX_MATCH_FIRST_LETTER = '/^[A-Za-z_]/';
+
+    private const REGEX_MATCH_STRING = '/^[A-Za-z0-9_]+$/';
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('kafka-schema-registry:check:template:name:field:all')
+            ->setDescription('Checks if name field follows avro naming convention')
+            ->setHelp('Checks if name field follows avro naming convention')
+            ->addArgument(
+                'schemaTemplateDirectory',
+                InputArgument::REQUIRED,
+                'Path to avro schema template directory'
+            );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        /** @var string $directory */
+        $directory = $input->getArgument('schemaTemplateDirectory');
+        $avroFiles = SchemaFileHelper::getAvroFiles($directory);
+
+        $io = new SymfonyStyle($input, $output);
+
+        $failed = [];
+
+        if (false === $this->checkSchemaTemplateNameFields($avroFiles, $failed)) {
+            $io->error('Following schema templates have invalid name field:');
```

Comment:
Maybe we can be a bit more descriptive and inform the "user" about the naming convention. E.g.
```
A template name must comply with the following rules:
- Start with a lower case letter
- End with a lower case letter or a number
- Contain only lower case letters and numbers and underscores
- Minimum length is 2 characters
The following template schema names violate the aforementioned rules:
- ...
```